datafusion/proto/proto/datafusion.proto

```diff
@@ -834,6 +840,7 @@ enum PrimitiveScalarType{
 
     BINARY = 25;
     LARGE_BINARY = 26;
+    FIXED_SIZE_BINARY = 29;
```

Comment:
I removed the duplication with PrimitiveScalarType and `ScalarValue` in https://github.com/apache/arrow-datafusion/commit/54d2870a56d8d8f914a617a7e2d52e387ef5dba2 so when you are resolving the merge conflict you should be able to just remove this code